### DISCUSSION TITLE
OpenDeck M stopped working

### DISCUSSION THREAD

**Discussion by squaretones** [shanteacontrols/OpenDeck#314] [Q&A] [answered]
Hi, I have an official OpenDeck M board. It suddenly stopped sending MIDI and is not being recognized by the configurator. I was planning on trying to reinstall the firmware. Does that make sense as a first step? If so, is opendeck31.uf2 file the correct firmware? 

I understand I have to "short" a pin on the board to put it into bootloader mode before installing the firmware. Which pin is that and how do I do that?

Thank you!

**Comment by paradajz** [comment, marked as answer]
> It suddenly stopped sending MIDI

During the use? During the configuration? On boot? Please add some details.

> I understand I have to "short" a pin on the board to put it into bootloader mode before installing the firmware. Which pin is that and how do I do that?

There's a header called RPB on the board (the right one):

![image](https://github.com/user-attachments/assets/cad39603-d0fe-48dd-b7d9-4268d6035585)

Connect it together before board startup and copy the new firmware on it. No, opendeck31.uf2 is not the right one because that one is for v3.1 board. The M board is v2.2 so you need opendeck22.uf2

**Reply by squaretones** [reply]
Thanks. 

> During the use? During the configuration? On boot? Please add some details. 

I had rewired a potentiometer connection and then plugged in the board to USB and opened the configurator to test it. It seemed to work for a second, but then then the configurator stopped recognizing the board at all and showed "No OpenDeck Board found". The LED on the board still lights up when plugged into USB. I tried different USB cables and USB ports and restarting the computer. I have yet to try disconnecting all components from the board. Is that worth doing?

**Reply by squaretones** [reply]
After I wrote that I realized I should probably try unplugging components and I did get it working again. After I wired the components back in it is still working fine. I assume I must have wired something wrong previously that caused it to stop working. Sorry for the false alarm and thanks again for the help.
